### DISCUSSION TITLE
Welcome to spring-ai-alibaba Discussions!

### DISCUSSION THREAD

**Discussion by chickenlj** [alibaba/spring-ai-alibaba#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yeohx** [comment]
我在查看https://java2ai.com/integration/mcps/mcp-overview mcp 协议文档的时候有一个疑问，mcp有服务端和客户端。为什么需要客户端呢？mcp服务端模型为什么不能直接访问呢？ 如果提供客户端。客户端通过function call的方式调用。岂不是多一个对话流程。因为function call需要模型返回调用方法和入参。然后本地代码调用具体的函数。既然mcp是对外公网可以访问。为什么模型不直接调用。为什么还需要mcp client呢？
